lib/realtime/tenants/migrations.ex

```diff
@@ -252,9 +254,9 @@ defmodule Realtime.Tenants.Migrations do
     Logger.info("Creating partitions for realtime.messages")
     today = Date.utc_today()
     yesterday = Date.add(today, -1)
-    tomorrow = Date.add(today, 1)
+    future = Date.add(today, 10)
```

Comment:
```suggestion
    future = Date.add(today, 3)
```

I do think we should keep this at 3 for now and setup a project on our end to continuously ping it so we can surface edge cases around this logic faster.